### PR TITLE
Removes documentation for deprecated API v1

### PR DESCRIPTION
**Pull request by nronnei**
Time to get it outta there.

### CODE REVIEW

**Comment by michaelambrose** [comment]
It's not depreciated yet.  I don't think we can remove this yet.  We still have lots of people using it!

**Comment by nronnei** [comment]
The documentation is still available; it won't be gone. There are still clear instructions to find it.

API v1 _is_ deprecated and no one new should use it. It's not been sunset yet.

**Comment by michaelambrose** [comment]
Gotcha, sounds good!

File: README.md [hunk 1]
```diff
@@ -5,6 +5,6 @@
 
 ## Versions
 
-[v1](/v1/README.md)
+~v1~ (DEPRECATED. If you need documentation, see the `deprecated` tag.)
```
Comment (nronnei) [inline README.md:8]: We can add a link to the `deprecated` tag to make sure it's even easier, if we like.

```suggestion
~v1~ (DEPRECATED. If you need documentation, see the [`deprecated`](https://github.com/docnetwork/api/tree/deprecated) tag.)
```

**Review by ConnorStec — APPROVED** [review]
Good by me. Docs are still accessible and we're no longer advertising a to-be-deprecated API in our newly-public repo.

**Pull request merged**